grammars.yml

```diff
@@ -316,9 +316,6 @@ vendor/grammars/fish-tmbundle:
 - source.fish
 vendor/grammars/forth:
 - source.forth
-vendor/grammars/fortran.tmbundle:
-- source.fortran
```

Comment:
It looks like the new grammar doesn't have this scope name, so you need to update `languages.yml` to point to the appropriate new scope name.